### PR TITLE
Add types

### LINKED ISSUE #541: Types For browserslist ?
would you be amenable to a PR with the types in this project ? (wanted to ask before I routed it) 

These ones are out of date.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/browserslist/index.d.ts

### PR DESCRIPTION
**Pull request by Semigradsky**
Fixes #541

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,10 +29,15 @@
     "node": "^6 || ^7 || ^8 || ^9 || ^10 || ^11 || ^12 || >=13.7"
   },
   "bin": "./cli.js",
+  "typings": "./index.d.ts",
```
Comment (ai) [inline package.json:32]: Can you rename to `types` to keep consistency across my projects?

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (ai) [inline tsconfig.json:1]: Do we need to add to `.npmignore`?

File: package.json (hunk 1 above)
Comment (Semigradsky) [reply package.json]: Sure

File: tsconfig.json (hunk 2 above)
Comment (Semigradsky) [reply tsconfig.json:1]: Sounds reasonable

**Comment by Semigradsky** [comment]
`update-db.test.js` test failed
https://travis-ci.org/github/browserslist/browserslist/jobs/739264898

**Comment by ai** [comment]
CI can be fixed in the latest `main`. Can you rebase PR (not that I changed the default branch from `master` to `main`)?

**Comment by Semigradsky** [comment]
Sure

**Comment by ai** [comment]
Released in 4.15.

**Pull request merged**